### ISSUE TITLE
contour overlay

### ISSUE THREAD

**Issue by jordandekraker** [khanlab/hippunfold-plot#7]
Something I was never able to get working with HippoMaps/BrainSpace's VTK plotting is adding contours as an overlay. This would be good for showing subfield outlines on a map, or for highlighting statistically significant areas. I'll look into this later, but I wanted to suggest it as a feature so we don't forget

**Comment by akhanf** [comment]
Yes thats a useful one, could use the approach taken here: https://nilearn.github.io/dev/modules/generated/nilearn.plotting.plot_surf_contours.html#nilearn.plotting.plot_surf_contours
